src/garage/tf/policies/discrete_qf_derived_policy.py

```diff
@@ -50,10 +50,8 @@ def get_action(self, observation):
                 len(self.env_spec.observation_space.shape):
             observation = self.env_spec.observation_space.unflatten(
                 observation)
-        q_vals = self._f_qval([observation])
-        opt_action = np.argmax(q_vals)
-
-        return opt_action, dict()
+        opt_actions, agent_infos = self.get_actions(np.array([observation]))
```

Comment:
why always the cast to np.ndarray?

Reply:
The method `get_actions` asks for a numpy array. I thought I would stay true to the documentation where possible.

I'll probably have to update the docs to allow for the union of all the different types of akro observations.

Reply:
...but why would observation not be a numpy array after flattening? wasn't that the point?